database/migrations/2018_06_27_080510_create_tasks_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTasksTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
Samim tim sto task pripada jednom user-u, nekako moramo da spojimo te tabele. 
Znaci na tabeli `tasks` treba da imas neki `user_id` 
`$table->unsignedInteger('user_id')`
Laravel generalno konta sa cim je to u vezi kad postavis relacije kako treba.